easybuild/easyconfigs/t/Tk/Tk-8.6.5-foss-2015b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.5'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
+ building a graphical user interface (GUI) in many different programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%(namelower)s%(version)s-src.tar.gz']
+
+patches = ['Tk-8.6.4_different-prefix-with-tcl.patch']
+
+dependencies = [
+    ('X11', '20160819'),
+    ('Tcl', version),
+    ('zlib', '1.2.8'),
+]
+
+configopts = '--enable-threads --with-tcl=$EBROOTTCL/lib CFLAGS="-I$EBROOTTCL/include" EXTRA_INSTALL="install-private-headers"'
+
+start_dir = 'unix'
+
+modextrapaths = {'TCLLIBPATH': 'lib/tk%(version_major_minor)s'}
```

Comment:
Same here, we also don't have this in other `Tk` easyconfigs, should we add this consistently?